src/Makefile.am

```diff
@@ -188,6 +188,10 @@ libsub_root_a_SOURCES = \
 	thread_base.cc \
 	thread_base.h \
 	thread_worker.cc \
-	thread_worker.h
+	thread_worker.h \
+	\
```

Comment:
Syntax error.

Reply:
Can you describe the syntax error for me? I'm unable to reproduce on a fresh checkout, and the same pattern is used farther up in the file.

Reply:
We don't need two forward slashes in a row. The compiler is just going to the next line twice. It won't throw an error, but should still fix it.

Reply:
I'm still confused as to the issue, I'm simply following the same pattern as above in the same file, where an extra line and forward slash is used to separate directories. Is it a syntax error, just a warning, or something else?